<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZhiXiangContract</class>
 <widget class="QWidget" name="ZhiXiangContract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1094</width>
        <height>1009</height>
       </rect>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>120</y>
         <width>391</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="Label_3">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>合同编号：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LineEdit_contract_id">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label_5">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>签订日期：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="dateEdit_qianding_date">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>431</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="1">
         <widget class="QLineEdit" name="LineEdit_xufang">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>需方：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>供方名称：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="LineEdit_gongfang">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>绍兴市图雨纺织有限公司</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>60</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>购 销 合 同</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="table_product">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>1041</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_jine_daxie">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>510</y>
         <width>551</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="Label_4">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>510</y>
         <width>211</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>金额合计（人名币大写）</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>550</y>
         <width>1041</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="Label_35">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1.产品规格特殊要求备注：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LineEdit_xieyi_1">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_36">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2.交货方式：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LineEdit_xieyi_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label_38">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3.付款方式：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="LineEdit_xieyi_3">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label_6">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4.交期</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="dateEdit_xieyi_4">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>545</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>一、产品名称、规格型号、数量、单位、金额：</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>700</y>
         <width>511</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>供  方</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_17">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>供  方：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LineEdit_gongfang_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label_18">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>经办人： </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="LineEdit_g_jingbanren">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label_19">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>地  址： </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="LineEdit_g_address">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="Label_20">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>电  话：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="LineEdit_g_telephone">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="Label_21">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手  机：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="LineEdit_g_mobilephone">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="Label_22">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>传  真：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="LineEdit_g_fax">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>700</y>
         <width>511</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>需  方</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_23">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>需  方：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LineEdit_xufang_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label_24">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>经办人：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="LineEdit_x_jingbanren">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label_25">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>地  址： </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="LineEdit_x_address">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="Label_26">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>电  话：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="LineEdit_x_telephone">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="Label_27">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手  机：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="LineEdit_x_mobilephone">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="Label_28">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>传  真：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="LineEdit_x_fax">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>960</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>添加合同</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>480</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>添加一行</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ZhiXiangContract</receiver>
   <slot>AddLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ZhiXiangContract</receiver>
   <slot>ButtonSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>985</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>991</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddLine()</slot>
  <slot>ButtonSlot()</slot>
 </slots>
</ui>
